### ISSUE TITLE
python3-pygments not found

### ISSUE THREAD

**Issue by drbeco** [slgobinath/libreoffice-code-highlighter#4]
I tried to install and got this error:

```
root@fox:~# pip3 install python3-pygments
Downloading/unpacking python3-pygments
  Could not find any downloads that satisfy the requirement python3-pygments
Cleaning up...
No distributions at all found for python3-pygments
Storing debug log for failure in /root/.pip/pip.log
root@fox:~#
```

The system in question is:

```
$ uname -a
Linux raposa 3.16.0-4-amd64 #1 SMP Debian 3.16.39-1+deb8u2 (2017-03-07) x86_64 GNU/Linux
```

Any clues on the causes?

**Comment by slgobinath** [comment]
Sorry, that's my mistake. It should be
```
pip3 install pygments
```

**Comment by slgobinath** [comment]
Fixed with https://github.com/slgobinath/libreoffice-code-highlighter/commit/bfe1ac55144ae95b26d48913cf18de7f5143bd24

**Comment by slgobinath** [comment]
Hi @drbeco,
I have released a new version tested with LO 5.3.2.2. If you are interested, please have a look at the Git Hub page. (README is updated)
